hexrd/ui/image_load_manager.py

```diff
@@ -207,7 +207,6 @@ def finish_processing_ims(self):
         if not self.update_status:
             self.update_needed.emit()
         if self.transformed_images:
-            HexrdConfig().instrument_config_loaded.emit()
```

Comment:
It looked like this had 2 effects:
1. Auto-generating cartesian and polar parameters
2. Calling `LoadPanel.config_changed()`, which only really does something if detectors were changed.

So, *hopefully* removing this won't have any negative side effects.